proposal/prefixed-parent-selector.md

```diff
@@ -0,0 +1,58 @@
+# Prefixed Parent Selector: Draft 2
+
+[Issue](https://github.com/sass/sass/issues/1425)
+
+## Background
+
+When transitioning from Less or Stylus, or just learning Sass, nesting is common.
+
+Everything works just as expected until a `&` is in the beginning of a nested selector.
+
+`"&" may only used at the beginning of a compound selector.` 
+
+Considering that there's been quite a few issues regarding this, and that they keep comming up, 
+and that people expect this to work, I think this would be a valuable addition to Sass.
+
+There's a work-around ([with some limitations](https://github.com/sass/sass/issues/1425#issuecomment-404462836)),
+but it's subpar. It might scare of beginners and/or cause unnecessary headache.
+
+
+## Summary
+
+Prefixed parent selectors will work as expected, solving the frustration of current, and new, users of Sass.
+
+@arkonan's [example](https://github.com/sass/sass/issues/1425#issuecomment-405921429) neatly summarises this: 
+```
+& selector { }  // works
+selector & { }  // works
+&selector { }   // works
+selector& { }   // doesn't work
+```
+
+
+
+## Syntax
```

Comment:
The syntax section should start with a [BNF](https://en.wikipedia.org/wiki/Backus%E2%80%93Naur_form)-style specification of the syntax of the new feature, including how it combines with existing syntax. You can look at other proposals for examples.

Reply:
The BNF thing is quite something. Doesn't seem like something you can just figure out and write an example for in 30 minutes. I'll need help with this, or I'm afraid this is where my energy runs out.

Reply:
Though, I followed my intuition and gave it a silly go; https://github.com/sass/sass/commit/031a6d6ca260b20155f028ee2dd8f232f1e976b5#diff-734ce04a16b4d09ecbe73e775f49927bR121